File: C program/areaofcircle.c
#include<stdio.h>
void main()
{
	float radius,area,circum;
	printf("Enter radius:");
	scanf("%f",&radius);
	area=3.14f*radius*radius;
	circum=2*3.14f*radius;
	printf("area=%f\n",area);
	printf("circum=%f",circum);
}

File: Csharp_programs/areaofcircle.cs
using System;

namespace areaofcircle
{
    internal class Program
    {
        public static void Main()
        {
            int radius;
            float area, circ;
            Console.WriteLine("Enter Radius : ");
            radius=Convert.ToInt32(Console.ReadLine());
            area = 3.14f * radius * radius;
            circ = 2 * 3.14f * radius;
            Console.WriteLine("Area = " + area);
            Console.WriteLine("Circumference = " + circ);
            Console.ReadKey();
        }
    }
}
